src/DataProtection/docs/Test-AzDataProtectionBackupInstanceRestore.md

```diff
@@ -0,0 +1,344 @@
+---
+external help file:
+Module Name: Az.DataProtection
+online version: https://docs.microsoft.com/powershell/module/az.dataprotection/test-azdataprotectionbackupinstancerestore
+schema: 2.0.0
+---
+
+# Test-AzDataProtectionBackupInstanceRestore
+
+## SYNOPSIS
+Validates if Restore can be triggered for a DataSource
+
+## SYNTAX
+
+### ValidateViaIdentity1 (Default)
+```
+Test-AzDataProtectionBackupInstanceRestore -InputObject <IDataProtectionIdentity>
+ -Parameter <IValidateRestoreRequestObject> [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm]
+ [-WhatIf] [<CommonParameters>]
+```
+
+### Validate1
+```
+Test-AzDataProtectionBackupInstanceRestore -Name <String> -ResourceGroupName <String> -VaultName <String>
+ -Parameter <IValidateRestoreRequestObject> [-SubscriptionId <String>] [-DefaultProfile <PSObject>] [-AsJob]
+ [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### ValidateExpanded1
```

Comment:
I think we only need to keep parameter set ValidateExpanded1

Reply:
removed extra param sets